src/biblebot/auth.py

```
@@ -0,0 +1,250 @@
+    path.write_text(json.dumps(creds.to_dict(), indent=2))
+    logger.info(f"Saved credentials to {path}")
+
+
+def load_credentials() -> Optional[Credentials]:
+    path = credentials_path()
+    if not path.exists():
+        return None
+    try:
+        data = json.loads(path.read_text())
+        return Credentials.from_dict(data)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)